src/dotty/tools/dotc/core/NaiveConstraint.scala

```diff
@@ -0,0 +1,383 @@
+package dotty.tools
+package dotc
+package core
+
+import Types._, Contexts._, Symbols._
+import util.SimpleMap
+import collection.mutable
+import printing.{Printer, Showable}
+import printing.Texts._
+import config.Config
+import config.Printers._
+
+object NaiveConstraint {
+
+  /** The type of `Constraint#myMap` */
```

Comment:
`Constraint#myMap` does not exist anymore, `NaiveConstraint#myMap` would be more correct.